graph/src/runtime/mod.rs

```diff
@@ -36,6 +36,18 @@ pub trait AscType: Sized {
     }
 }
 
+impl<T> AscType for std::marker::PhantomData<T> {
+    fn to_asc_bytes(&self) -> Result<Vec<u8>, DeterministicHostError> {
+        Ok(vec![])
+    }
+
+    fn from_asc_bytes(asc_obj: &[u8]) -> Result<Self, DeterministicHostError> {
+        assert!(asc_obj.len() == 0);
```

Comment:
Should this be a `DeterministicHostError`?

Reply:
I agree that this is an assert.